jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -1620,11 +1620,8 @@ BoundMethodHandle rebind() {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
 	
-/*[IF Sidecar19-SE-OpenJ9]*/
 	void customize() {
```

Comment:
Pls add a comment to the code to explain this implementation.

Reply:
Added a comment.
In addition, modified `JPP` flag `Sidecar19-SE-OpenJ9` to `Sidecar18-SE-OpenJ9` such that the change only applies to `OpenJ9`.
Please have another look.

Reply:
Perhaps the comment should mention the test which needs this stub.

Reply:
I didn't feel to state explicitly that this code is added to pass a test. How about adding a link to the issue in the comment instead?

Reply:
ok

Reply:
Added a link to the issue. Please review.